SeleniumWebDriverBasics/YandexUnitTests.cs

```diff
@@ -0,0 +1,252 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Support.UI;
+using static System.Net.WebRequestMethods;
+
+namespace SeleniumWebDriverBasics
+{
+    [TestFixture]
+    public class YandexUnitTests
+    {
+        private IWebDriver driver;
+        private string baseUrl;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            var service = FirefoxDriverService.CreateDefaultService();
+            this.driver = new FirefoxDriver(service);
+            this.driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            this.baseUrl = "https://mail.yandex.com/";
+            this.driver.Navigate().GoToUrl(this.baseUrl);
+            this.driver.Manage().Window.Maximize();
+        }
+
+        [Test]
+        public void LoginToYandex()
+        {
+            //Arrange
+            var expectedAccountName = "katetest.selenium";
+
+            //Act
+            LoginToYandexMail();
+
+            IsElementVisible(By.XPath("//span[text()='katetest.selenium']"));
+            var actualAccountName = this.driver.FindElement(By.XPath("//span[text()='katetest.selenium']")).Text;
+
+            //Assert
+            Assert.AreEqual(expectedAccountName, actualAccountName);
+        }
+
+        [Test]
+        public void CreateDraftEmail()
+        {
+            //Arrange
+            var expectedSubject = "Test Selenium";
+
+            //Act
+            LoginToYandexMail();
+
+            IsElementVisible(By.XPath("//a[@href='#compose']"));
+
+            this.driver.FindElement(By.XPath("//a[@href='#compose']")).Click();
+            IsElementVisible(By.XPath("//div[@class='composeYabbles']"));
+            this.driver.FindElement(By.XPath("//div[@class='composeYabbles']")).SendKeys("testkate837@gmail.com");
+            this.driver.FindElement(By.XPath("//input[@class='composeTextField ComposeSubject-TextField']")).Click();
+            this.driver.FindElement(By.XPath("//input[@class='composeTextField ComposeSubject-TextField']")).SendKeys("Test Selenium");
+            this.driver.FindElement(By.XPath("//div[@class='cke_wysiwyg_div cke_reset cke_enable_context_menu cke_editable cke_editable_themed cke_contents_ltr cke_htmlplaceholder']")).SendKeys("Test Selenium from Kate");
+            this.driver.FindElement(By.XPath("//a[@href='#draft']")).Click();
+            IsElementVisible(By.XPath("//span[@title='Test Selenium']"));
```

Comment:
As steps for draft email creation are used in several tests, they can be moved into a separate CreateDraftEmail method to avoid code duplication. The same for sending email.